.github/workflows/dart.yml

```diff
@@ -1,9 +1,10 @@
-# Created with package:mono_repo v3.1.0-beta.2
+# Created with package:mono_repo v3.1.0-beta.3
 name: Dart CI
 on:
   push:
     branches:
-      - $default-branch
+      - main
```

Comment:
What is this doing?

Reply:
`$default-branch` is only valid for **templates** – learned that the hard way

Defaulting these to master & main to cover the common default cases.

Unlikely someone will have both. Eliminates the need for a lot of custom logic/config

Reply:
I don't understand, isn't that what this file is? The existing config is working...

Reply:
Synced off-thread. LGTM.